bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/binding/ThingHandler.java

```diff
@@ -174,4 +175,18 @@
      */
     void handleRemoval();
 
+    /**
```

Comment:
Adding this hint in the JavaDoc would require an additional import also in the MANIFEST (at least, Eclipse wanted to have that. I don't think, that's worth mentioning it. Maybe if I use the full qualified name ... will try